packaging/rpm/microshift.spec

```diff
@@ -242,7 +242,7 @@ systemctl enable --now --quiet openvswitch || true
 %{_bindir}/cleanup-all-microshift-data
 %{_unitdir}/microshift.service
 %{_sysconfdir}/crio/crio.conf.d/microshift.conf
-%{_sysconfdir}/microshift/config.yaml
+%{_sysconfdir}/microshift/config.yaml.default
```

Comment:
```suggestion
%config(noreplace) %{_sysconfdir}/microshift/config.yaml
```

Reply:
see:

https://docs.fedoraproject.org/en-US/packaging-guidelines/#_configuration_files
https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html

Reply:
When we make changes to the config file /etc/microshift/config.yaml.rpmnew

Will be written into disk for user reference, but the user changes will be kept in disk